server/src/main/java/org/elasticsearch/search/aggregations/bucket/range/AbstractRangeBuilder.java

```diff
@@ -44,7 +46,8 @@
     protected boolean keyed = false;
 
     protected AbstractRangeBuilder(String name, InternalRange.Factory<?, ?> rangeFactory) {
-        super(name, rangeFactory.getValueType());
+        // TODO: We used to set targetValueType here to rangeFactory.getValueType(), not sure if we need that information now.
```

Comment:
This is one of the cases where we seemed to be doing something important with targetValueType, and I didn't see a direct way to capture that information in the new framework.  I ended up using it for the default values source type, below, but I'm not at all sure this is correct.

Reply:
Hmm.  So I think the only places this affects things are stand-alone scripts, unmapped fields, and formatting for both... so the default down below should be equivalent?  I think this is just an artifact of how the Range agg is structured, sharing base classes but trying to specialize parsing and output with subclassing.